Makefile

```
@@ -79,7 +79,7 @@ test:
 
 copy:
 	@echo "copying binaries..."
-	@mkdir -p ./hack/bin
+	@mkdir -p ./hack/bine
```

```suggestion
	@mkdir -p ./hack/bin
```

this seems to be a typo :-)